### PR TITLE
Set timeout for making call to database

### PR DESCRIPTION
**Pull request by Pradhyumna02**
Set timeout value for a database call. As the test it getting struck when we don't get response back from the server.

### CODE REVIEW

File: public/examples/latency/latencyApp.js [hunk 1]
```diff
@@ -164,7 +164,7 @@
 
     function latencyBasedRouting() {
         // pass in the client location instead of the hard coded value
-        var latencyBasedRouting = new window.latencyBasedRouting('NJ', '/testServer', latencyBasedRoutingOnComplete, latencyBasedRoutingOnError);
+        var latencyBasedRouting = new window.latencyBasedRouting('NJ', '/testServer', 2000, latencyBasedRoutingOnComplete, latencyBasedRoutingOnError);
```
Comment (rotosonic) [inline public/examples/latency/latencyApp.js]: let make 2000 a variable

**Comment by rotosonic** [comment]
1+ once hardcoded timeout set to variable tested with no data and data returned

File: public/lib/latencyBasedRouting.js [hunk 2]
```diff
@@ -25,14 +25,15 @@
      * @param callbackComplete - callback function for test suite complete event
      * @param callbackError - callback function for test suite error event
```
Comment (awangc) [inline public/lib/latencyBasedRouting.js]: Need to update the comments to include `timeout`

File: public/examples/latency/latencyApp.js (hunk 1 above)
Comment (awangc) [inline public/examples/latency/latencyApp.js]: agreed

**Comment by awangc** [comment]
Had one additional comment. Provisional :+1: given comments are addressed

**Comment by rotosonic** [comment]
1+

**Comment by awangc** [comment]
:+1:

**Pull request merged**